docs/security.md

```diff
@@ -1,15 +1,17 @@
 Security
 ==========
 
-Applications built on Apex run as native YARN applications on Hadoop. The security framework and apparatus in Hadoop apply to the applications. The default security mechanism in Hadoop is Kerberos.
+Applications built on Apex run as native YARN applications on Hadoop. The security framework and apparatus in Hadoop apply to the applications. Both authentication and SSL aspects of the security framework are covered here.
 
```

Comment:
SSL as we are supporting currently is for stram web services isn't it. For example, this is not the SSL configuration for RPC or when stram is a web services client. I would consider moving it to web services security.